armory/scenarios/base.py

```diff
@@ -0,0 +1,102 @@
+import abc
+import argparse
+import json
+import logging
+import os
+import time
+
+import coloredlogs
+
+import armory
+from armory import paths
+from armory.utils import config_loading
+
+
+logger = logging.getLogger(__name__)
+
+
+class Scenario(abc.ABC):
+    def __init__(self):
+        pass
+
+    def validate_config(self, config):
+        """
+        Validate the scenario config
+        """
```

Comment:
What would be done in here that is different from our config file validation?

Reply:
This was for any scenario-specific validation. Perhaps that can be handled when instantiating the subclasses.

Reply:
Better handled at subclass instantiation.